### PR TITLE
3.0/bug/filters noshow

### PR DESCRIPTION
**Pull request by mikael-stromgren**
- Local react files instead of CDN
- Several errors in Gutenberg fixed
- Filter functionality restored
- Design fixes (dependant of classes added in hbg-react)

### CODE REVIEW

File: source/js/Module/Event/Components/FilterableEventsContainer.js [hunk 1]
```diff
@@ -214,13 +215,15 @@ class FilterableEventsContainer extends React.Component {
...
+      post_id: postId,
+      settings: JSON.stringify(settings),
       page: currentPage,
       per_page: perPage,
       search_string: searchString,
       start_date: startDate,
       tags,
     };
-
+    console.log(settings);
```
Comment (faejr) [inline source/js/Module/Event/Components/FilterableEventsContainer.js:226]: leftover console.log

File: source/php/Helper/React.php [hunk 2]
```diff
@@ -4,20 +4,29 @@
...
+        $version = '17.0.1';
 
         //Enqueue react
-        if (!wp_script_is('react')) {
-            wp_enqueue_script('react', 'https://unpkg.com/react@' . $version . '/umd/react.' . $suffix . '.js', array(), null);
-        }
+        wp_enqueue_script('react', EVENTMANAGERINTEGRATION_URL.'/source/js/front/react/react.' . $suffix  . '.js', array(), null);
+        wp_enqueue_script('react-dom', EVENTMANAGERINTEGRATION_URL.'/source/js/front/react/react-dom.' . $suffix  . '.js', array(), null);
+        //wp_enqueue_script('react', 'https://unpkg.com/react@' . $version . '/umd/react.' . $suffix . '.js', array(), null);
+        //wp_enqueue_script('react-dom', 'https://unpkg.com/react-dom@' . $version . '/umd/react-dom.' . $suffix . '.js', array(), null);
```
Comment (faejr) [inline source/php/Helper/React.php:28]: if this is no longer necessary it should be removed

**Review by faejr — CHANGES_REQUESTED** [review]
Looks fine, but I've never worked with this part so I'm not entirely sure. 

Other than the comments I wonder if it's possible to enqueue react without keeping these huge react files in the repo? Could we get them with npm or something instead?

File: source/js/Module/Event/Components/FilterableEventsContainer.js (hunk 1 above)
Comment (mikael-stromgren) [reply source/js/Module/Event/Components/FilterableEventsContainer.js]: done

File: source/php/Helper/React.php (hunk 2 above)
Comment (mikael-stromgren) [reply source/php/Helper/React.php]: and done

**Pull request merged**